staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -80,6 +80,8 @@ type Connection struct {
 	// Presence when the type is "direct" will also cause an error
 	// +optional
 	HTTPConnect *HTTPConnectConfig `json:"httpConnect,omitempty"`
+
+	UDSName string `json:"udsName,omitempty"`
```

Comment:
UDS is not required, so this should be optional. Its adding a new field, so it should be optional.